Hunting/Assets/Scripts/GameManager/SunController.cs

```diff
@@ -10,34 +10,94 @@ public class SunController : MonoBehaviour
     private float dayTime = 60;
 
     [SerializeField]
-    private float dimFactor = 3;
+    private float currentTimeOfDay = 0.5f;
+
+    public float secondsInFullDay = 120f;
+    public float timeMultiplier = 0.5f;
+
+    public MeshRenderer cloud;
     
     # endregion
 
     # region Fields
 
-    private float moveAngle = 0;
 
-    private new Light light = null;
+    private new Light sun;
+    private new float sunInitialIntensity;
+
+    private new Material high = null;
+
+    private new float r = 1f;
+    private new float g = 1f;
+    private new float b = 1f;
+
+
+    public System.Collections.Generic.List<Color> colors = new System.Collections.Generic.List<Color>();

```

Comment:
Same thing as above, you should make this field private. If you want to expose this field in the Unity editor and use the colour-picker to select the colours that you want, remember to use the [SerializeField] attribute (I have noticed that you set the colour in your code though).